lib/googmodule.js

```diff
@@ -24,14 +24,15 @@ var createPreprocesor = function(logger, /* config.basePath */ basePath) {
 
     log.debug('Processing "%s".', file.originalPath);
 
-    // Path must include http:// protocol and '/base/' so that Karma can still recognize the URL
-    // and apply source maps.
-    content =
-        JSON.stringify(content + '\n//# sourceURL=http://googmodule/base/' + relativePath + '\n');
-    // Make sure not to insert any line breaks before the content, so that line
```

Comment:
Seems like this aspect of the previous version is not retained?

Reply:
Correct. Per my comment above, the source displayed in the debugger contains the wrapper either way, so since this wasn't doing anything I removed it.